baselines/Google Chat Minimum Viable Secure Configuration Baseline v0.1.md

```diff
@@ -300,3 +300,72 @@ Agencies SHOULD configure DLP rules to block or warn on sharing files with sensi
 9.  Select the appropriate action to warn or block sharing, based upon the agency's individual requirements.
 10. In the **Alerting** section, choose a severity level, and optionally, check **Send to alert center to trigger notifications**.
 11. Review the rule details, mark the rule as **Active**, and click **Create.**
+
+## 7. Content Reporting
+
+This section covers the content reporting functionality.
```

Comment:
Recommend expanding this to define what content reporting is. It's hard to tell exactly what it is just from the name and not all users will be familiar with it.

```suggestion
This section covers the content reporting functionality, a feature that allows users to report messages that violate organizational guidelines to workspace admins.
```